exporter/awsemfexporter/metric_translator.go

```diff
@@ -80,6 +80,7 @@ type groupedMetricMetadata struct {
 	logGroup                   string
 	logStream                  string
 	metricDataType             pmetric.MetricType
+	batchIndex                 int
```

Comment:
What is this used for?

Reply:
EMF exporter aggregate incoming metrics into groupedMetrics, where the groupKey is generated based on the groupedMetricMetadata. After splitting, the two new split will share exactly the same key. Here I add an extra metric metadata for key generation, this new entry won't be added into emf log, and it can prevent the second metric from dropping.